migrations/dashboard.sql

```diff
@@ -13,6 +13,7 @@ CREATE TABLE IF NOT EXISTS incidents (
   approved   BOOLEAN DEFAULT FALSE,
   emu        BOOLEAN DEFAULT FALSE,
   active     BOOLEAN DEFAULT FALSE,
+  crd        BOOLEAN DEFAULT FALSE,
```

Comment:
I don't think this code will be executed on existing deployments. Supposedly you'll also need an `ALTER TABLE … ADD COLUMN …` line below.

Reply:
yes will do